go/oasis-test-runner/oasis/network.go

```diff
@@ -657,9 +655,6 @@ func (net *Network) startOasisNode(
 		cfg.Consensus.StateSync.TrustHeight = node.consensusStateSync.TrustHeight
 		cfg.Consensus.StateSync.TrustHash = node.consensusStateSync.TrustHash
 	}
-	if viper.IsSet(metrics.CfgMetricsAddr) {
-		extraArgs = extraArgs.appendNodeMetrics(node)
```

Comment:
How does this work for tests now? During tests we want to be able to collect metrics from all nodes and also the test runner itself.

For example, see [the daily txsource test](https://github.com/oasisprotocol/oasis-core/blob/master/.buildkite/scripts/daily_txsource.sh).

Reply:
Oh, damn, that's actually used...  I'll fix the test-runner to take cmdline args again and convert them into the config struct internally.